src/modules/Tab/TabGroup.story.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import Tab from "./Tab";
+import TabGroup from "./TabGroup";
+
+interface IProps {
```

Comment:
Do we really need to declare these props? Can we import from the TabGroup file?

Reply:
That's a good point 👍